WalletWasabi/Extensions/MemoryExtensions.cs

```diff
@@ -41,7 +41,8 @@ public static async Task<TItem> AtomicGetOrCreateAsync<TItem>(this IMemoryCache
 			{
 				if (!cache.TryGetValue(key, out value))
 				{
-					value = await cache.GetOrCreateAsync(key, factory).ConfigureAwait(false);
+					value = await factory.Invoke();
```

Comment:
Is it by design that there is no `.ConfigureAwait(false)`?

Reply:
Fixed.